Zend/zend_API.c

```diff
@@ -426,9 +426,20 @@ ZEND_API bool ZEND_FASTCALL zend_parse_arg_class(zval *arg, zend_class_entry **p
 }
 /* }}} */
 
+static ZEND_COLD bool zend_null_arg_deprecated(const char *type) {
+	zend_string *func_name = get_active_function_or_method_name();
+	zend_error(E_DEPRECATED, "%s(): Passing null to argument of type %s is deprecated",
```

Comment:
Shouldn't it say "Passing null as argument" or "Passing null to parameter"?

Reply:
I've changed it to say "Passing null to parameter".

Reply:
I decided to thread through the argument number after all, and provide a more detailed error message:

> Deprecated: date_interval_create_from_date_string(): Passing null to parameter #1 ($datetime) of type string is deprecated